### PR TITLE
implement additional document pages

### PR DESCRIPTION
**Pull request by CslBing**

### CODE REVIEW

File: server/constants/paths.ts [hunk 1]
```diff
@@ -12,6 +12,14 @@ const paths = {
     CONTACT_DETAILS: '/order/:orderId/about-the-device-wearer/contact-details',
     RESPONSIBLE_ADULT: '/order/:orderId/about-the-device-wearer/responsible-adult',
   },
+
+  ATTCHMENT: {
```
Comment (ben-cart3r) [inline server/constants/paths.ts:16]:
```suggestion
  ATTACHMENT: {
```

File: server/views/pages/order/attachments/edit.njk [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+    errorMessage:error
+  }) }}
+
+  <div class="govuk-button-group">
+    {{ govukButton({
+        text: "Save and continue",
+        type: 'submit',
+        value: "continue"
+    }) }}  
+    <a class="govuk-link" href="/order/{{ orderId }}/attachments">Return to order section menu</a>
```
Comment (ben-cart3r) [inline server/views/pages/order/attachments/edit.njk:44]: I think the text here is wrong

File: server/views/pages/order/attachments/view.njk [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+     <dd class="govuk-summary-list__value">
+       No photo id document uploaded
+    </dd>   
+    {% endif %}
+    <dd class="govuk-summary-list__actions">
+      <a class="govuk-link" href="/order/{{order.id}}/attachments/photoId">Change<span class="govuk-visually-hidden"> name</span></a>
+    </dd>
+  </div>
+</dl>
+
```
Comment (ben-cart3r) [inline server/views/pages/order/attachments/view.njk:80]: Should there be a next button / return to summary button

**Review by ben-cart3r — DISMISSED** [review]
👍

File: server/views/pages/order/attachments/edit.njk (hunk 2 above)
Comment (CslBing) [reply server/views/pages/order/attachments/edit.njk]: updated to say Return to attachment summary page

File: server/views/pages/order/attachments/view.njk (hunk 3 above)
Comment (CslBing) [reply server/views/pages/order/attachments/view.njk]: added Back to form section button

**Review by ben-cart3r — APPROVED** [review]
👍

**Pull request merged**